src/exporter.js

```diff
@@ -0,0 +1,174 @@
+const sha1 = require('sha1');
+
+export default class {
+  constructor(deckName) {
```

Comment:
`sql`, because `require` is environment specific. 

Sorry for misdirection, I realized I was using the GitHub review feature incorrectly which led to a lot of scattered and unconnected comments. Now I am going to stick to a single review with "changes requested" status.

```js
// index.js
export default function(deckName) {
  return new ApkgExporter(deckName, { sql, template });
}
// exporter.js
constructor() {
  ...
  this.db = new sql.Database();
  ...
}
```